environment.yml

```
@@ -0,0 +1,17 @@
+name: chemprop
+channels:
+  - pytorch
+  - conda-forge
+dependencies:
+  - python>=3.11
+  - pytorch::pytorch>=2.1
+  - pytorch::torchaudio
+  - pytorch::torchvision
+  - astartes[molecules]
```

```suggestion
  - astartes
```

The `astartes[molecules]` package is only for PyPI - to get it on `conda`, you just do `conda install astartes aimsim`, which we have accomplished here by specifying `aimsim` in the next line.